pennylane_lightning/src/bindings/Bindings.cpp

```diff
@@ -19,8 +19,14 @@
 
 #include "pybind11/pybind11.h"
 
-#if defined(_ENABLE_PLQUBIT) || _ENABLE_PLKOKKOS == 1
-
+// Defining the module name.
+#if defined(_ENABLE_PLQUBIT)
+#define pennylane_lightning_ops lightning_qubit_ops
```

Comment:
Macros definitions should always be UPPER_CASE. We can also call this something more succinct like LIGHTNING_MODULE_NAME